base/applications/network/netstat/netstat.c

```diff
@@ -98,6 +98,10 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                         // UNIMPLEMENTED.
                         ConPuts(StdErr, L"'b' option is FIXME (Accepted option though unimplemented feature).\n");
                         bDoShowProcName = TRUE;
+
+                        // 2k3sp2 and XPSP3: -b does imply -o (superior)
+                        // Win7 and 8.1: -b and -o were decoupled (inferior)
```

Comment:
Not sure having "superior" or "inferior" helps in anything here.
You could instead say that we follow what netstat does on Windows <= 2003, and that it is more logical to have -b show the processes IDs, since -b itself shows the process image that created the connection.

(Also, comment style for non-temporary/FIXMEs/TODOs here is with `/* ... */`).

Reply:
Would some `#if _WIN32_WINNT` or the likes be applicable here?

Reply:
Yes, using `#if _WIN32_WINNT` macros would be preferable.

Reply:
To be honest, I don't like the fact that the ifdef will automatically switch back to the inferior state when ros will update that define in several years to target more recent versions of Windows. But if that is the preference of the majority of devs then I can do that.

Reply:
Fixed heavy-heart-edly.